### PR TITLE
fix: use applicationuserlistQuery from ChannelSetting

### PR DESCRIPTION
**Pull request by sravan-s**
* use applicationuserlistQuery from ChannelSetting
* some visual issues in User Panel

### CODE REVIEW

File: src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import '../../../../styles/variables';
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss:1]: Also scroll in popup

[the remark this answers is not in the document]
Comment (HoonBaek) [reply src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss:1]: image
Why this doesn't look same current state?

[the remark this answers is not in the document]
Comment (sravan-s) [reply src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss:1]: this is not same screen ~

[the remark this answers is not in the document]
Comment (HoonBaek) [reply src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss:1]: Aha!

**Review by HoonBaek — APPROVED** [review]

File: src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (sravan-s) [reply src/smart-components/ChannelSettings/components/UserPanel/user-panel.scss:1]: but, I need to probably change the color of badge and add little top-padding

**Comment by sravan-s** [comment]
Screen Shot 2023-03-03 at 1 08 25 PM

**Pull request merged**